README.md

```diff
@@ -1 +1,122 @@
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+
+# 📖 **HELLO MICROVERSE PROJECT** <a name="about-project"></a>
+
+**Hello Microverse Project. Jose RM.** In this project, I set up a "Hello world" repository. I applied the tools and best practices that I learned about in previous steps.
+
+## 🛠 Built With <a name="Visual Studio Code">HTML5 & CSS3</a>
+
+### Tech Stack <a name="Visual Studio Code"></a>
+
+### Key Features <a name="key-features"></a>
+
+- HTML5
+- CSS5
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+Clone the repository using git, and open it with visual studio code.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+  cd "your-folder"
+  git clone git@github.com:https://github.com/joramot/hello-world.git
+
+
+### Usage
+
+To open the project, use yout prefered explorer and open the index.html file 
+
+
+### Deployment
+
+You can deploy this project using visual studio.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 ** Jose RM **
+
+- GitHub: https://github.com/joramot
+- Twitter: https://twitter.com/el_rulis
+- LinkedIn: https://linkedin.com/in/jose-RM
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project you can contact the author througt the above media links or by email to mojica.raul@gmail.com
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse and all the persons that got in touch with me while we were working on this project
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ❓ FAQ <a name="faq"></a>
+
+- Is it difficult?
+
+  - If you have knowledge about github basic commands, installing npm extension in visual studio code, figma, HTML and CSS it is easier, in other way, it can be little complicated
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Please make sure to include a LICENSE to your project, at the moment this is not working. Microverse suggests you use the MIT one. Follow the [official docs](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to achieve this! 🤓

Reply:
Thank Andres, I am going to work on this issue in a moment.